components/resource-creation/ResourceSearchSort.tsx

```diff
@@ -1,87 +1,47 @@
 import { Button, Group, TextInput } from '@mantine/core';
 import { IconChevronDown, IconChevronUp, IconSearch, IconSelector } from '@tabler/icons';
-import React, { useEffect, useRef, useState } from 'react';
-import { getFhirResourceSummary } from '../../util/fhir/codes';
+import React from 'react';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { cardFiltersAtom } from '../../state/atoms/patientResources';
 
-type Props = {
-  resources: fhir4.BundleEntry[];
-  onSorted: (sortedResources: fhir4.BundleEntry[]) => void;
-  dateForResource: (resource: fhir4.FhirResource) => { date: string; dateType: string };
-};
+const ResourceSearchSort = () => {
+  const setCardFilters = useSetRecoilState(cardFiltersAtom);
+  const cardFilters = useRecoilValue(cardFiltersAtom);
```

Comment:
```suggestion
  const [cardFilters, setCardFilters] = useRecoilState(cardFiltersAtom);
```
You can simplify these to one line (will also need to update recoil imports).